app/lib/Plugins/SearchEngine/ElasticSearch/FieldTypes/Intrinsic.php

```diff
@@ -118,7 +118,7 @@ public function getIndexingFragment($pm_content, $pa_options) {
 			$this->getTableName() . '/' . $this->getFieldName() => $pm_content
 		);
 
-		if($t_instance->getProperty('ID_NUMBERING_ID_FIELD') == $this->getFieldName() || (is_array($pa_options) && in_array('INDEX_AS_IDNO', $pa_options))) {
+		if($t_instance->getProperty('ID_NUMBERING_ID_FIELD') == $this->getFieldName() || (is_array($pa_options) && in_array('INDEX_AS_IDNO', $pa_options, true))) {
```

Comment:
Does this make sense? The config is parsed like as follows:
```
array:3 [
  "BOOST" => "100"
  0 => "DONT_TOKENIZE"
  1 => "TOKENIZE"
]
```
Without this fix entity labels are exploded into their individual parts which makes it impossible to match exact strings with spaces.

So the search clause ca_entity_labels:"Department of Obscure Things" would not match a record where the associated entity is "The Department of Obscure Things".